### ISSUE TITLE
User interface improvements

### ISSUE THREAD

**Issue by Garuda-1** [google/robotstxt-spec-test#7]
## TL;DR
Current user interface lacks some significant information that could help to debug the tested parser/matcher.

## Useful information to provide
From my experience, it would be great to provide:
  - Parser output
  In case of exception, return code may be altered from ALLOWED and DISALLOWED patterns. Thus, the only information potential user sees is UNSPECIFIED outcome. This may occur either if the provided command is invalid or if runtime error occurred. So, at least standard output would help a lot to determine cause of UNSPECIFIED outcome.
  - Test cases identification
  In some tests items of ```test_expectations``` list may not be visibly identified, which makes error identification complicated. For example, if one of line endings tests fails, it's hard to determine only by robots.txt contents whether CR, CRLF, LF or mixed line endings are not handled properly. Possible solution could be avoiding similar ```additional_explanation``` field values or adding ```tag``` field.
  - Progress bar
  Testing Java parser takes 1.5 minutes, and if it follows the standard, no output will be produced until all tests are passed. It's important to see testing progress. Significant delays may indicate low performance or the fact that some tests cause infinite loops.

## Minor propositions
  - Command
  It may be difficult to create a valid command to call the parser / matcher (e.g. tricky special characters escape rules). Giving a passed command would be useful for debugging.
  - Differencing standard and Google-specific optimisations
  The fact that parser FOLLOWS / DOES NOT FOLLOW the __standard__ based on the CTC is wrong. Some test cases validate Google-specific behaviour which is not related to the standard. It would be more precise to say that the parser FOLLOW / DOES NOT FOLLOW the __standard and google-specific optimisations__.
